jpa/platform-tests/pom.xml

```diff
@@ -23,14 +23,14 @@
         <groupId>jakarta.tck</groupId>
         <artifactId>persistence-tck</artifactId>
         <version>3.2.0</version>
-        <relativePath>../pom.xml</relativePath>
     </parent>
     <artifactId>persistence-tck-platform-tests</artifactId>
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
             <groupId>jakarta.servlet</groupId>
             <artifactId>jakarta.servlet-api</artifactId>
+            <version>6.0.0</version>
```

Comment:
6.1.0-M2 is the latest for EE1

Reply:
6.1.0 is the latest at the staging, hence added same.